frontend/wallet/src/render/components/SettingsPage.re

```diff
@@ -109,20 +110,20 @@ let make = () => {
   let dropDownValue =
     switch (networkValue) {
     | NetworkOption(s) => Some(s)
-    | Custom(_) => Some("CUSTOM")
+    | Custom(_) => Some("Custom network")
```

Comment:
qq - do we ever use constants for string literals? makes sense if used in multiple places like this one - to avoid repeating it / making a mistake - also allows us to change copy in one place if we ever decide to

Reply:
Yeah that's probably a good idea, for things like this it'd be best to just create a variable, but in this case it's simple enough we can probably get away with it.